ensemble-view/src/main/java/hec/ensembleview/DatabaseHandlerService.java

```diff
@@ -63,6 +62,10 @@ public void setDbHandlerZonedDateTime(ZonedDateTime zdt) {
         }
     }
 
+    public boolean getIsIssueDateAvailableForDss() {
```

Comment:
Having 'DSS' in the method name raises some questions:

* Is the DatabaseHandlerService intended as a interface? 
* What is different besides the class EnsembleDatabase interface